resources/clj-kondo.exports/lilactown/helix/clj_kondo/lilactown/helix.clj

```diff
@@ -0,0 +1,70 @@
+(ns clj-kondo.lilactown.helix
+  (:require [clj-kondo.hooks-api :as api]))
+
+#_{:clj-kondo/ignore [:clojure-lsp/unused-public-var]}
+(defn $
```

Comment:
making a note that we should do the same for `helix.dom/$d`. doesn't have to be a part of this PR

Reply:
wtf is `$d`? 

There's no docstring, but it looks like it does "DOM props coercion". Are you thinking that doing it at that fn's definition will make it work for all `d/*`?

Reply:
@riotrah `$d` is for creating react elements for DOM nodes. previously, `$` was meant to special case "native" elements like DOM elements for react-dom, however the heuristic it used isn't good ([i talk about that in this blog post](https://lilac.town/writing/towards-helix-1-0/#native-element-creation-macros)) so I created `$d` to handle that logic instead, and plan on deprecating it in `$`

Reply:
i don't think that kondo macroexpands so i am guessing it won't extend to all of `d/*`

Reply:
There's a different way coming up from me. 

https://clojurians.slack.com/archives/CHY97NXE2/p1652120240410379?thread_ts=1651602537.647199&cid=CHY97NXE2

But it looks like maybe in the future clj-kondo might support an easier syntax anyways